azure-pipelines.yml

```diff
@@ -313,8 +313,11 @@ jobs:
           AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
           GOOGLE_CLOUD_PROJECT: $(GOOGLE_CLOUD_PROJECT)
           GOOGLE_APPLICATION_CREDENTIALS: $(ciGCSServiceAccountKey.secureFilePath)
+          AZCOPY_E2E_TENANT_ID: $(OAUTH_TENANT_ID)
```

Comment:
The reason the test sees the env variable as nil is because this set of env variables apply to the smoke tests, not the UT tests. To set for the UT tests, see run-ut.yml in the azurePipelineTemplates directory.